botorch/models/converter.py

```diff
@@ -248,6 +258,18 @@ def batched_to_model_list(batch_model: BatchedMultiOutputGPyTorchModel) -> Model
             )
         if isinstance(batch_model, SingleTaskMultiFidelityGP):
             kwargs.update(batch_model._init_args)
+        # NOTE: Adding outcome transform to kwargs to avoid the multiple
+        # values for same kwarg issue with SingleTaskMultiFidelityGP.
+        if outcome_transform is not None:
+            octf = outcome_transform.subset_output(idcs=[i])
+            kwargs["outcome_transform"] = octf
+            # Update the outcome transform state dict entries.
+            sd = {
+                **{k: v for k, v in sd.items() if "outcome_transform" not in k},
+                **{"outcome_transform." + k: v for k, v in octf.state_dict().items()},
```

Comment:
I don't think it's necessary to exclude `"outcome_transform"` in the first line here - should be overwritten by the content of the state dicts?

Reply:
Good point!